### PR TITLE
[apis] Update Readme.md with content in the google doc provided

### PR DESCRIPTION
**Pull request by nicoto**
# Summary

Fixes issue #467

Here's a description of changes:

* ...
* ...
* ...

_Before you merge, make sure you visually check your changes in the deploy preview. A link to the preview will appear in the PR comments when ready._

cc @gdisf/admins

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview ready!

Built with commit bdcf76b8505313efcffa6062ba6c7af80e7ef1d6

https://deploy-preview-490--teaching-materials.netlify.com

File: apis/README.md [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
+
+Prerequisites
+
+Attendees should be comfortable with HTML and CSS fundamentals (HTML tags and attributes, CSS selectors) and JavaScript fundamentals (arrays, conditionals, loops, functions). Attendees should also be comfortable with jQuery. That means that you attended our Intro to HTML & CSS workshop, Intro to JavaScript workshop, and Intro to jQuery workshop, or you learned those concepts online.
 
 You can review them online at a few places:
-* HTML/CSS: [Khan Academy's HTML/CSS course](https://khanacademy.org/html-css), [Codecademy's HTML/CSS course](https://www.codecademy.com/tracks/web), [GDI SF HTML/CSS slides](http://teaching-materials.org/htmlcss-1day)
-* JS: [Codecademy's JS course](https://www.codecademy.com/tracks/javascript), [Khan Academy's Intro to JS course](https://khanacademy.org/programming),  [GDI SF JS slides](http://teaching-materials.org/javascript)
-* jQuery: [Codecademy's jQuery course](https://www.codecademy.com/tracks/jquery)
+HTML & CSS: Khan Academy's HTML & CSS course, Codecademy's HTML & CSS course, GDI SF's HTML & CSS slides
```
Comment (trucnguyen) [inline apis/README.md]: Could you preserve the markdown links? :)

File: apis/README.md [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
+JS widget APIs
+JSONP APIs
+JS API libraries
 
-Attendees should be comfortable with HTML/CSS basics and JavaScript fundamentals (arrays, conditionals, loops, functions). Attendees should also have learned jQuery. That means that you either attended our HTML 101 workshop, JS 101 workshop and JQuery workshop, or that you learned those concepts online. 
+Slides
+
+http://teaching-materials.org/apis/ 
+
+Prerequisites
```
Comment (trucnguyen) [inline apis/README.md]: Please keep the markdown formatting

**Comment by nicoto** [comment]
JS303 Intro to client-side APIs

**Comment by trucnguyen** [comment]
Thanks!

**Comment by trucnguyen** [comment]
@nicoto I have given you another raffle ticket for this PR

**Pull request merged**